### DISCUSSION TITLE
How can I see the stream_executor logs of xla?

### DISCUSSION THREAD

**Discussion by knightXun** [openxla/xla#16442] [Q&A]
I want to see the steam_executor logs to diagnosis a certain bug. I already know `TF_CPP_MIN_LOG_LEVEL=0 TF_CPP_MAX_VLOG_LEVEL=2` output the full xla logs, but it make too many noise.

**Comment by mooskagh** [comment]
You can specify vlog level by module, e.g. `TF_CPP_VMODULE=stream_executor=2`.

**Reply by mooskagh** [reply]
Just to confirm, you set is as environment variable, not passing as the flag, right?

https://github.com/pytorch/xla/blob/master/TROUBLESHOOTING.md says you'd also need `TF_CPP_MIN_LOG_LEVEL=0`, but otherwise it should work.

**Reply by knightXun** [reply]
I use these flags like this: `TF_CPP_MIN_LOG_LEVEL=0 TF_CPP_MAX_VLOG_LEVEL=2 TF_CPP_VMODULE=stream_executor=10 python nmnist_exodus_snn.py`
